src/themes/shared/typography.ts

```diff
@@ -94,12 +94,6 @@ export const textStyles = {
     lineHeight: lineHeights.sm,
     fontSize: fontSizes.base,
   },
-  'button-sm': {
```

Comment:
I checked with the UX-Designer and we won't need the button-small token for future designs. We are good to remove it. Thank you!